### DISCUSSION TITLE
Angular Support

### DISCUSSION THREAD

**Discussion by rebound-software** [meteor/meteor#12895] [Feature Requests]
Are there any plans to support Angular >v13 development for Meteor once v3.0 is here?

I have a significant amount of development into a project based on Angular with Meteor. Angular has had some significant updates since v13 and I was hoping Meteor v3 might remove some of the barriers that currently prevent anything beyond v13 working (as I understand is the case)?

Trying to decide if I backed the wrong horse and port to another framework (React/Vue etc.) or stick with Angular in the hope that support for Angular will eventually improve...

**Comment by StorytellerCZ** [comment]
Not sure what the issues are exactly. If people can document them here then that will make it easier for the team to address them. I personally couldn't get into Angular in the past so I don't have the necessary knowledge to offer any help here. But I think after 3.0 is released things like this should hopefully become easier.
